INSTALL_DOCKER.rst

```diff
@@ -80,37 +80,13 @@ To update your `Clear Linux`_ installation to the latest execute:
 
     sudo swupd update
 
-Uninstall container-basic
-~~~~~~~~~~~~~~~~~~~~~~~~~
-
-If you are on an older version of `Clear Linux`_ you may have an old version of the ``container-basic`` bundle installed, that contains an older version of Docker_. If so, remove this bundle:
-
-  ::
-
-    sudo swupd bundle-remove containers-basic
-
 Install Clear Containers bundles
```

Comment:
This should now be titled, "Install Clear Containers bundle" as only 1 bundle now needs to be installed.